querydsl-sql/src/test/java/com/querydsl/sql/SelectWindowFunctionsBase.java

```diff
@@ -27,7 +24,7 @@
 public class SelectWindowFunctionsBase extends AbstractBaseTest {
 
     @Test
-    @ExcludeIn(SQLSERVER) // FIXME
+    @ExcludeIn({SQLSERVER, NUODB}) // FIXME
```

Comment:
Don't include this class in the NuodbSuite, if window functions are not supported